docs/getting_started.rst

```diff
@@ -139,9 +139,31 @@ Alternatively, to create just the output file or directory of one rule, run:
 
      snakemake -j<NUMBER_OF_CPU_CORES> <output file or folder>
 
+If needed, all calculated scenarios in the directory :file:`results` can be deleted with the following rule:
+
+::
+
+    snakemake -j1 clean
```

Comment:
As soon as https://github.com/rl-institut/oemof-B3/pull/329 is merged, you'll need to merge dev into this branch and adapt this info distinguishing between linux and windows operating system. :)

Reply:
Thank you @MaGering for the helpful review so far! 

As the description for the clean rule needs to move to PR #325, adapting the overall information for the `clean` rules have to be done in the branch features/add_empty_ts. Is this correct to assume?

Reply:
Excuse me, I confused the two snakemake rules.